internal/core/item_tree.rs

```diff
@@ -745,6 +754,18 @@ impl ItemRc {
         result
     }
 
+    pub fn window_item(&self) -> Option<Self> {
```

Comment:
This function is fairly specific to `resolve_font_property` which itself already has a loop. I think the two loops could be combined in one.
(i.e: i'd move this function out of ItemRc and move it in `resolve_font_property`, or make it non-revursive and just re-use the loop that is already in `resolve_font_property`)
Otherwise, please document.